### PR TITLE
'What is a Blockchain' Article

### PR DESCRIPTION
**Pull request by RichieJohn**

### CODE REVIEW

File: content/dogepedia/articles/what-is-a-blockchain.md [hunk 1]
```diff
@@ -5,4 +5,10 @@ date = "2021-10-22"
...
+A blockchain is a digital distributed ledger that is used as a system for validating and recording transaction information. This digital ledger of transactions is duplicated and distributed throughout an entire network of computers connected to the blockchain.  
+
+The Dogecoin blockchain consists of thousands of computers – called [nodes](/dogepedia/articles/what-is-a-node) – meaning that transaction information is not stored in one centralised place. Instead, Dogecoin blockchain information is stored across thousands of nodes, and is why cryptocurrencies like Dogecoin are often referred to as being *decentralised*. 
+
+When a Dogecoin transaction is made, the information regarding the transaction is stored in what is a called a **block**, which is then distributed to the other nodes throughout the block**chain**. Every block that is created is then **chained** to the preceding block, so on and so forth all the way back to Dogecoin’s genesis block. 
+
+In addition to being peer-to-peer and decentralised, another benefit of blockchain technology is that is extremely difficult — and often impossible — to hack, alter, or deceive the system.
```
[the remark this answers is not in the document]
Comment (MishaBoar) [reply content/dogepedia/articles/what-is-a-blockchain.md:14]: I am not sure if this is too detailed for the level of depth we want to get to with these articles. Also, I would like @michilumin and the others to take a look at this before we push it live. Simple does not mean it has to be conceptually misleading.

**Review by MishaBoar — CHANGES_REQUESTED** [review]
I proposed some changes to add the concept of miners, clarify how blocks are chained, and some details about why attacks to the network are unlikely.

I would love if some of the Core devs could review this article. Even though it is very ELI5, we must make sure the condensed information is still "overall" correct.

File: content/dogepedia/articles/what-is-a-blockchain.md [hunk 2]
```diff
@@ -5,4 +5,10 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-A fairly accessible introduction to what a blockchain is.
+A blockchain is a digital distributed ledger that is used as a system for validating and recording transaction information. This digital ledger of transactions is duplicated and distributed throughout an entire network of computers connected to the blockchain.  
+
+The Dogecoin blockchain consists of thousands of computers – called [nodes](/dogepedia/articles/what-is-a-node) – meaning that transaction information is not stored in one centralised place. Instead, Dogecoin blockchain information is stored across thousands of nodes, and is why cryptocurrencies like Dogecoin are often referred to as being *decentralised*. 
+
+When a Dogecoin transaction is made, the information regarding the transaction is stored in what is a called a **block**, which is then distributed to the other nodes throughout the block**chain**. Every block that is created is then **chained** to the preceding block, so on and so forth all the way back to Dogecoin’s genesis block.
```
[the remark this answers is not in the document]
Comment (rnicoll) [reply content/dogepedia/articles/what-is-a-blockchain.md:12]: Can we avoid the timestamp reference? Miners give it a timestamp according to their own clock, and the network then goes "Eh, close enough" (or not), so it's definitely not an exact timestamp.

Actually I think the original text is closer, but let me give it a go:

"Every block that is created contains the hash of the preceding block, which in turn contains the hash of its preceding block, so on and so forth creating a chain all the way back to Dogecoin’s genesis block."

Notably the blockchain goes backwards only, and finding the next block is something a node handles as part of its implementation detail.

File: content/dogepedia/articles/what-is-a-blockchain.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rnicoll) [reply content/dogepedia/articles/what-is-a-blockchain.md:14]: Do we maybe want to have a separate page on 51% attacks, and introduce that later, rather than touch on them briefly here? @michilumin ?

File: content/dogepedia/articles/what-is-a-blockchain.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MishaBoar) [reply content/dogepedia/articles/what-is-a-blockchain.md:12]: Hey Ross, thank you! I edited the suggestion to include your suggestions!

File: content/dogepedia/articles/what-is-a-blockchain.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mishaboar2) [reply content/dogepedia/articles/what-is-a-blockchain.md:14]: Yeah, maybe this is out of place here? @RichieJohn can completely ignore my suggestion!

[the remark this answers is not in the document]
Comment (MishaBoar) [reply content/dogepedia/articles/what-is-a-blockchain.md:14]: @RichieJohn for this iteration, I think we can ignore this suggestion and use your version as per @rnicoll's suggestion. A separate article for 51% attacks will do.

File: content/dogepedia/articles/what-is-a-blockchain.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MishaBoar) [reply content/dogepedia/articles/what-is-a-blockchain.md:12]: Hey @RichieJohn - the suggested edits above include the changes Ross recommended.

File: content/dogepedia/articles/what-is-a-blockchain.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RichieJohn) [reply content/dogepedia/articles/what-is-a-blockchain.md:14]: OK - great. Thanks, @MishaBoar!

**Review by MishaBoar — APPROVED** [review]
Looks good to me!
